### DISCUSSION TITLE
Cloning speed is very slow

### DISCUSSION THREAD

**Discussion by laduniestu** [shivammathur/homebrew-php#2009] [Q&A / Help] [answered]
Hi, i try to run `brew install shivammathur/php/php@7.4`
but the cloning speed is very slow, right now currently in china.
is there any influence because of my region right now?
because i've check my connection is good, i can see youtube video in 4k without any problems
<img width="562" alt="image" src="https://github.com/shivammathur/homebrew-php/assets/44757637/fa9c804d-48b2-41c8-8ce6-05107d1a1b7e">

**Comment by shivammathur** [comment, marked as answer]
You are cloning the core repo that has all the core homebrew formulae. Since this is a large repo, this process is slow. Yes, the exact speed of cloning would depend on your region and internet connection.

To install PHP it should not clone the core repo with the latest version of brew. Please try upgrading it and then installing PHP again.

```zsh
brew upgrade
brew install shivammathur/php/php@7.4
```
